tck/features/clauses/return-orderby/ReturnOrderBy6.feature

```diff
@@ -0,0 +1,90 @@
+#
+# Attribution Notice under the terms of the Apache License 2.0
+#
+# This work was created by the collective efforts of the openCypher community.
+# Without limiting the terms of Section 6, any Derivative Work that is not
+# approved by the public consensus process of the openCypher Implementers Group
+# should not be described as “Cypher” (and Cypher® is a registered trademark of
+# Neo4j Inc.) or as "openCypher". Extensions by implementers or prototypes or
+# proposals for change that have been documented or implemented should only be
+# described as "implementation extensions to Cypher" or as "proposed changes to
+# Cypher that are not yet approved by the openCypher community".
+#
+
+#encoding: utf-8
+
+Feature: ReturnOrderBy6 - Aggregation expressions in order by
+
+  Scenario: [1] Handle constants and parameters inside an order by item which contains an aggregation expression
+    Given an empty graph
+    And parameters are:
+      | age | 38 |
+    When executing query:
+      """
+      MATCH (person)
+      RETURN avg(person.age) AS avgAge
+      ORDER BY $age + avg(person.age) - 1000
+      """
+    Then the result should be, in any order:
+      | avgAge |
+      | null |
+    And no side effects
+
+  Scenario: [2] Handle variables, which are also return items themselves, inside an order by item which contains an aggregation expression
+    Given an empty graph
+    When executing query:
+      """
+      MATCH (me: Person)--(you: Person)
+      RETURN me.age AS age, count(you.age) AS cnt
+      ORDER BY age, age + count(you.age)
+      """
+    Then the result should be, in any order:
+      | age | cnt |
+    And no side effects
+
+  Scenario: [3] Handle property accesses, which are also return items themselves, inside an order by item which contains an aggregation expression
+    Given an empty graph
+    When executing query:
+      """
+      MATCH (me: Person)--(you: Person)
+      RETURN me.age AS age, count(you.age) AS cnt
+      ORDER BY me.age + count(you.age)
+      """
+    Then the result should be, in any order:
+      | age | cnt |
+    And no side effects
+
+  Scenario: [4] Throw if a variable is used inside an order by item which contains an aggregation expression
+    Given an empty graph
+    When executing query:
+      """
+      MATCH (me: Person)--(you: Person)
+      RETURN count(you.age) AS agg
+      ORDER BY me.age + count(you.age)
+      """
+    Then a SyntaxError should be raised at compile time: AmbiguousAggregationExpression
+
+  Scenario: [5] Throw if more complex expression, even though it is also a return item on its own, is used inside an order by item which contains an aggregation expression
+    Given an empty graph
+    When executing query:
+      """
+      MATCH (me: Person)--(you: Person)
+      RETURN me.age + you.age, count(*) AS cnt
+      ORDER BY me.age + you.age + count(*)
+      """
+    Then a SyntaxError should be raised at compile time: AmbiguousAggregationExpression
```

Comment:
```suggestion
    Then a SyntaxError should be raised at compile time: AmbiguousAggregationExpression

```

Empty line at the end. More a random convention than a true necessity. But lets stick with it.

Reply:
Hm, I have added an extra line, but it doesn't show in the changed files. Is it there? Can you see it 😕

Reply:
It does not show empty lines at the end. But it shows a 
<img width="164" alt="image" src="https://user-images.githubusercontent.com/30618026/168232472-4c85d719-4b38-4a36-8ad6-bfaff1b6b404.png">
That is how I knew, empty line were missing. If this symbol is not shown, then everything is good.